### PR TITLE
Can't log into youtrack. Simple changes to repositoryfactory and umbr…

### PR DESCRIPTION
**Pull request by lars-erik**
…aco.web.ui assemblyinfo.

### CODE REVIEW

File: src/Umbraco.Web/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -34,5 +34,6 @@
 [assembly: InternalsVisibleTo("Concorde.Sync")]
 [assembly: InternalsVisibleTo("Umbraco.Courier.Core")]
 [assembly: InternalsVisibleTo("Umbraco.Courier.Persistence")]
+[assembly: InternalsVisibleTo("Umbraco.VisualStudio")]
```
Comment (Shazwazza) [inline src/Umbraco.Web/Properties/AssemblyInfo.cs]: If we do this here, we should do this for the Umbraco.Core one too right?

Comment (lars-erik) [inline src/Umbraco.Web/Properties/AssemblyInfo.cs]: It's already there.

Comment (lars-erik) [inline src/Umbraco.Web/Properties/AssemblyInfo.cs]: There's a dependency on an internal class from Umbraco.Web.UI in PetaPocoRepositoryBase.

Comment (Shazwazza) [inline src/Umbraco.Web/Properties/AssemblyInfo.cs]: Seems good to me then, i'll pull this in.

**Pull request merged**